unitary/alpha/quantum_world.py

```diff
@@ -97,17 +125,23 @@ def peek(
            equal to the count parameter.  Each element will be a list
            of measurement results for each object.
         """
+        if _num_reps is not None:
```

Comment:
nit:
Why the double negative logic? It took me a little bit of extra braining to parse. _num_reps is optional, so will most  likely be None. Seems more straight forward, and easier for me to parse, to be 

if _num_reps is None:
  ...
else
  ...

Reply:
Done.